docs_source_files/content/introduction/on_test_automation.de.md

```diff
@@ -8,9 +8,9 @@ weight: 2
 auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
 zu übersetzen indem Du uns einen Pull Reqeust schickst!
  {{% /notice %}}
-First, start by asking yourself whether or not you really need to use a browser.
+First, start by asking yourself whether or not you need to use a browser.
 Odds are that, at some point, if you are working on a complex web application,
-you will need to open a browser and actually test it.
+you will need to open a browser and test it.
```

Comment:
I think the purpose of those words was to set a tone where the reader should reflect if they need WebDriver or not. Let's leave it like it is.

Reply:
Reverted!